assignment/assignment-8-timed/assignment-8.2.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment-8.2</title>
+    </head>
+    <body>
+        <form>
+            <h1>Simple Quiz</h1>
+            <div>
+                <label for="fullNameInput">Full Name</label>
+                <input type="text" name="fullName" id="fullNameInput" />
+
+                <label for="emailInput">Email</label>
+                <input
+                    type="email"
+                    name="email"
+                    id="emailInput"
+                    placeholder="example@gmail.com"
+                />
+            </div>
+            <div>
+                <label for="confidenceLevelInput"
+                    >Rate Your Confidence Level</label
+                >
+                <input
+                    type="range"
+                    name="confidenceLevel"
+                    id="confidenceLevelInput"
+                    min="1"
+                    max="10"
+                />
+                <label for="ageInput">Your Age</label>
+                <input
+                    type="number"
+                    name="age"
+                    id="ageInput"
+                    min="18"
+                    max="99"
+                />
+            </div>
+            <h2>Select your favorite quiz categories</h2>
+            <div>
+                <label for="historyInput">History</label>
```

Comment:
use Checkbox to end of for and id attrs when using checkbox input
So should be historyCategoryCheckbox for id and for attrs

Reply:
Good catch. Thanks